src/main/java/uk/gov/hmcts/reform/sendletter/services/ReportsService.java

```diff
@@ -45,6 +46,7 @@ public ReportsService(
         this.timeToHour = downtimeFromHour;
     }
 
+    @Transactional
```

Comment:
should we not have an integration test which will add dummy records to db as part of setup and retrieve records from db and verify?

Reply:
@nitinprabhu added integration test for the reports endpoint.